src/commands/help.js

```diff
@@ -0,0 +1,35 @@
+const Discord = require('discord.js')
+module.exports = {
+  prefix: '!help',
+  /**
+   * @name help
+   * Displays currently available commands.
+   *
+   * @param {Discord.Message} message the message provided
+   */
+
+  /**
+   *Use this format to add new commands:
+   *{name: "Command Name", value: "Command Description"},
+   */
+
+  command: function help(message) {
```

Comment:
Formatting for this code doesn't look right, as the indentation is off. Are you running this code in an editor that can format with prettier on save? Once automated linting is setup this will error out.

Reply:
Ran it through prettier!

Reply:
Ran through prettier